docs/src/changelog.md

```diff
@@ -25,6 +25,10 @@ If your currently installed *Quality-time* version is not the latest version, pl
 - The software documentation was outdated (among other things the API-server health check endpoint). Fixes [#10858](https://github.com/ICTU/quality-time/issues/10858).
 - Keep the footer at the bottom of the page even if the report is very short. Fixes [#10877](https://github.com/ICTU/quality-time/issues/10877).
 
+### Removed
+
+- Don't install fonts in the renderer image, but let Chromium use its default fonts for PDF exports. This reduces the renderer image size with about 44MB. Closes [#10835](https://github.com/ICTU/quality-time/issues/10835).
```

Comment:
```suggestion
- Don't install fonts in the renderer image, but let Chromium use its default fonts for PDF exports. Closes [#10835](https://github.com/ICTU/quality-time/issues/10835).
```

Reply:
I like having the reason in the change log so it's easier to remember why we removed the fonts. I can make it shorter though:
```suggestion
- To reduce the size of the renderer image, don't install fonts and let Chromium use its default fonts for PDF exports. Closes [#10835](https://github.com/ICTU/quality-time/issues/10835).
```